test/input_gen/genModelTests_v2.py

```diff
@@ -985,3 +1005,17 @@ def forward(self, inputs, labels):
 
     #    Function to check the created golden test file
     inspect_file("non_trainable_fc_idx3.nnmodelgolden")
+
+    # Add ChannelShuffle test
+    channel_shuffle = ChannelShuffle(split_number=4)
+    record_v2(
+        channel_shuffle,
+        iteration=2,
+        input_dims=[(1, 8, 4, 4)],  # batch_size=1, channels=8, height=4, width=4
+        input_dtype=[float],
+        label_dims=[(1, 8, 4, 4)],
+        name="channel_shuffle",
+    )
+
+    #    Function to check the created golden test file
+    inspect_file("channel_shuffle.nnmodelgolden")
```

Comment:
How about packing the unittest into `unittest_models_v2.tar.gz` instead of `unittest_models_v3.tar.gz`? It seems that basic layer unit tests are currently using V2.